jetbrains-core/tst/software/aws/toolkits/jetbrains/core/stack/PageButtonsTest.kt

```diff
@@ -2,8 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package software.aws.toolkits.jetbrains.core.stack
 
-import assertk.assert
-import org.junit.Assert
```

Comment:
we should create a ktlint rule that detects junit `Assert` (which is an abomination against mankind).

Reply:
We use it in few places so we need to fix those first